content/en/tutorials/artifacts.md

```diff
@@ -443,7 +443,7 @@ class ConvNet(nn.Module):
 ```
 
 Here, we're using W&B to track the run,
-and so using the [`wandb.config`](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/wandb-config/Configs_in_W%26B.ipynb)
+and so using the [`run.config`](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/wandb-config/Configs_in_W%26B.ipynb)
```

Comment:
It's not letting me make a suggestion here but you could lose the comma and the "so".